code/modules/chemistry/tools/food_and_drink.dm

```diff
@@ -767,6 +767,13 @@ ABSTRACT_TYPE(/obj/item/reagent_containers/food/snacks)
 			else
 				boutput(user, "<span class='alert'>There's nothing in the bowl to dip!</span>")
 
+		else if (istype(W, /obj/item/reagent_containers/food/snacks/churro))
```

Comment:
this could just be an or case on the if above, like `else if (istype(...) || istype(...))`